### ISSUE TITLE
Not Updating Theme from Github

### ISSUE THREAD

**Issue by tregenza** [afragen/github-updater#683]
I have a theme on github [ https://github.com/tregenza/Tregenza-One ] which was installed via Github Updater. However GHU is not updating the installed version of the theme on my test Wordpress installation.

Current version of the theme on my Test installation is 0.5.2 and the last release on Github was 0.5.5, two days ago.

How do I go about solving the problem?

**Comment by tregenza** [comment]
Sorry about that. Not sure quite how it happened but somehow my development and GitHub repositories got out of sync and nothing had been committed.

However I have fixed the problem and the theme is still not updating.

I've created a new tag 0.5.6 (with matching style.css) and double checked the release contains the right style.css.

I've refreshed the cache and monitored the wp_cron activity. A ghu_get_remote_theme with no arguments is firing but nothing is being updated.

There is nothing in debug.log to indicate any problem.

**Comment by afragen** [comment]
Does the theme show up in the GitHub tab on the Settings page?

Can you screenshot that page?

**Comment by afragen** [comment]
I just installed the theme and was able to update, reinstall etc.

**Comment by tregenza** [comment]
Sorry the delay, I'm juggling projects at the moment.

The theme is showing in the Github tab (see screeenshot) everything looks OK. Cron jobs are running when I refresh the tabs.

I've check file permissions and everything has write permissions. I can install and update Wordpress and themes / plugins via Wordpress and I originally installed the theme via GHU so I assume it has all the right permissions.  

![screen shot 2018-05-17 at 14 13 34](https://user-images.githubusercontent.com/550563/40179731-eb4f3d3a-59dc-11e8-863a-f6ebd00cd8fc.png)

**Comment by tregenza** [comment]
This is the cron job which fires when I refresh the cache.

![screen shot 2018-05-17 at 14 30 57](https://user-images.githubusercontent.com/550563/40180895-c96b6aec-59df-11e8-94e2-a0f699364bc2.png)

**Comment by tregenza** [comment]
Think I might have spotted the problem - seem to be calling the development branch.

Will investigate and let you know.

**Comment by afragen** [comment]
You can use the branch switcher to see what branch is current and to switch or reinstall a branch. For plugins that’s on the plugins.php page and for themes it’s on the themes.php page.

**Comment by afragen** [comment]
On a single site you need to select the individual theme card. The branch switching option is display there.

**Comment by tregenza** [comment]
Problem solved!

In the end I deleted the theme and reinstalled it. It now works perfectly - detecting and installing changes on Github appropriately. 

I suspect the problem was a combination of unknowing being on the wrong branch and some changes to my firewall I made as I tried to identify/fix the issue.

I've updated the GHU Wiki to include a bit more information where the Branch Switcher can be found (it really isn't obvious on Single User installs). 

And I've just logged an new change request issue to provide better information for the user on the Github Updater screen and better debug logging. It would have saved me several hours of fiddling around. 

Thanks for all the help.
